pkg/debug/scripts/collect-info.sh

```diff
@@ -47,11 +48,12 @@ usage()
     echo "Read-logs mode:"
     echo "       -d                   - read device logs only"
     echo "       -a APPLICATION-UUID  - read specified application logs only"
+    echo "       -t NUMBER-OF-DAYS    - read logs from the last NUMBER-OF-DAYS [1-30]"
```

Comment:
Why is it capped at 30?

Reply:
just a number, i thought if we are that far way, why not just get all the logs